src/components/FSHControls.js

```diff
@@ -11,6 +13,10 @@ import DialogContentText from '@material-ui/core/DialogContentText';
 import DialogTitle from '@material-ui/core/DialogTitle';
 import { runSUSHI, runGoFSH } from '../utils/FSHHelpers';
 import { sliceDependency } from '../utils/helpers';
+import { TreeView, TreeItem } from '@material-ui/lab';
+import CodeMirrorComponent from './CodeMirrorComponent';
+import AssignmentOutlinedIcon from '@material-ui/icons/AssignmentOutlined';
+import LibraryBooksIcon from '@material-ui/icons/LibraryBooks';
```

Comment:
This is really small, but can you put these new material UI imports up with the existing material UI imports.

Reply:
Absolutely!

Reply:
Addressed in commit `3c148e3`